.github/workflows/qa.yaml

```diff
@@ -231,7 +231,7 @@ jobs:
         working-directory: ${{ matrix.subproject }}
         run: |
           go test -shuffle=on ./... -race -tags=gowslmock
-      #- name: Upload coverage to Codecov
-      #  uses: codecov/codecov-action@v3
-      #  with:
-      #    file: /tmp/coverage.out
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          file: /tmp/coverage.out
```

Comment:
The input option is `files` :p

https://github.com/codecov/codecov-action?tab=readme-ov-file#arguments

Reply:
Interesting, we are still using the `file:` in adsys which definitively works (https://github.com/ubuntu/adsys/blob/main/.github/workflows/qa.yaml#L80), but probably an initial option that has been deprecated and then unlisted. Let’s use the new format then!